### PR TITLE
Remove billing support email

### PR DESCRIPTION
**Pull request by jingcheng16**
Ticket: https://dimagi-dev.atlassian.net/browse/SAAS-14988

##### Environments Affected

All env will be affected: Staging, India, Swiss, Production

This is already deployed to staging.

I will run update-config, and restart service, to india, swiss and production, after this PR is approved and merged.

### CODE REVIEW

File: environments/64-test/public.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ data_email: droberts+datatree@dimagi.com
 subscription_change_email: droberts+accounts+subchange@dimagi.com
 internal_subscription_change_email: droberts+accounts+subchange+internal@dimagi.com
 billing_email: droberts+billing-comm@dimagi.com
-invoicing_contact_email: droberts+billing-support@dimagi.com
+invoicing_contact_email: droberts+accounts@dimagi.com
```
Comment (millerdev) [inline environments/64-test/public.yml:81]: FYI @dannyroberts, you may need to update mail filters for this.

**Review by millerdev — APPROVED** [review]

**Comment by gherceg** [comment]
> I will run update-config, and restart service, to india, swiss and production

You may already know this but honestly this is also to clarify my own understanding - I think `service commcare restart` will need to be scheduled during maintenance hours for production (via the maintenance feed). I believe it is true that this command doesn't disrupt services since it is a rolling restart that doesn't impact formplayer machines, but as an extra precaution we still want to schedule it during maintenance hours. @dannyroberts or @millerdev does that sound right to you?

File: environments/64-test/public.yml (hunk 1 above)
Comment (dannyroberts) [reply environments/64-test/public.yml:81]: Thanks. But also, eh, this "environment" is defunct anyway. Can probably just remove it entirely.

**Comment by dannyroberts** [comment]
I'll offline with you @gherceg about rollout procedures

**Pull request merged**